x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/dashboards/task/dashboard_migrations_telemetry_client.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { DashboardMigrationDashboard } from '../../../../../common/siem_migrations/model/dashboard_migration.gen';
+import {
+  SIEM_MIGRATIONS_MIGRATION_ABORTED,
+  SIEM_MIGRATIONS_MIGRATION_FAILURE,
+  SIEM_MIGRATIONS_MIGRATION_SUCCESS,
+  SIEM_MIGRATIONS_RULE_TRANSLATION_FAILURE,
+  SIEM_MIGRATIONS_RULE_TRANSLATION_SUCCESS,
```

Comment:
It is not a big deal but dashboard reporting `RULE` events is misleading.

Either we can remove this file or i am okay tackling this later as well. commenting here so that we know this exist.

Reply:
Fixed here https://github.com/elastic/kibana/pull/234046/commits/aba7d70bca5c3fe339d561fb36078c228ab55192

I updated some `eventName` values like:

`Translation success` -> `Rule translation failure`

@logeekal I assumed `evenNames` are only used to display the name in the dashboards, changing them won't cause any issue, is that correct?

Reply:
>hey are not used to query/filter

Not they are not. Mainly used for display only. Although, i will double check. If there is occurrence, i will change it.